### ISSUE TITLE
go.layout.Updatemenu restyle does not resize the jupyter output_wrapper 

### ISSUE THREAD

**Issue by Krulvis** [plotly/plotly.py#2206]
When using updatemenus to resize a plot, the output window of a jupyter notebook does not resize along with the plot. This causes a scrollbar to appear while if you would set the size of the plot to the larger variant, the cell's output window would be scaled along with the size of the plot. The height is controller and set statically in the 'plotly-graph-div' element along with the 'svg-container' element. After a restyle the 'svg-container' gets changed (obviously to alter the size of the graph), while the 'plotly-graph-div' stays the same.

Is there a way to make sure that the 'plotly-graph-div' changes along with a size restyle of a graph from within the cell's code? After I run the cell, I can manually change the 'height' style tag of the 'plotly-graph-div' element to 'auto' and it will work but this is not the expected outcome and shouldn't be necessary.

Code to reproduce issue (ran in a notebook ofcourse):

```
import plotly.graph_objs as go

x=[0,1,2,3]
y=[1,2,3,4]

traces = [go.Bar(
    x=x,
    y=y
)] 

layout = go.Layout(
    updatemenus=[go.layout.Updatemenu(
        x=0.1,
        y=1.2,
        active=0,
        showactive=True,
        buttons=list([
            dict(label="X - Y",
               method="relayout",
               args=[
                      {"width": 1000,
                      "height": 400}
               ]),
            dict(
               label="Y - X",
               method="relayout",
               args=[
                      {"width": 1000,
                      "height": 700}
               ])
        ])
    )]
)
fig = go.Figure(data=traces, layout=layout)
fig.show()
```

tl;dr The cell's height is controller by in the 'plotly-graph-div' element inside the output_wrapper. It is set to a static height on creation and does not change after a 'restyle' update.

**Comment by gvwilson** [comment]
Hi - we are currently trying to tidy up Plotly's public repositories to help us focus our efforts on things that will help users most. Since this issue has been sitting for several years, I'm going to close it. If it's still a concern, we'd be grateful if you could open a new issue (with a short reproducible example if appropriate) so that we can add it to our backlog. Thanks for your help - @gvwilson
